README.rst

```diff
@@ -120,6 +122,12 @@ requires a minimal settings file for Django-required configurations.
    cp ci/testsettings.py.sample testsettings.py
    python -c "import uuid; print('\nSECRET_KEY = \'%s\'' % uuid.uuid4())" >> testsettings.py
 
+- By default, parasolr expects Solr 8. If running tests with an earlier 
+  version of Solr, either explicitly change **MAJOR_SOLR_VERSION** in your
+  local **testsettings.py** or set the environment variable::
+
+   export SOLR_VERSION=x.x.x
+
```

Comment:
I was probably unclear in my suggestions about how to implement this, sorry.

I'd like to see the Solr version as an optional config setting in the `SOLR_CONNECTIONS` dict. 

For the tests on travis-ci, my expectation is that the python config would need to be populated from an environment variable since we can't hard-code it in test settings.

Reply:
... but maybe this doesn't actually matter, if the only changes are specific to how we test?

Reply:
From conversation during pair: keeping this for now since it's only for testing and easily changeable wihtin the local `testsettings.py`